src/cryptonote_core/blockchain.cpp

```diff
@@ -2670,15 +2670,15 @@ static bool get_fcmp_tx_tree_root(const BlockchainDB *db, const cryptonote::tran
 
   // Make sure reference block exists in the chain
   CHECK_AND_ASSERT_MES(tx.rct_signatures.p.reference_block < db->height(), false,
-      "tx included reference block that was too high");
+      "tx " + epee::string_tools::pod_to_hex(get_transaction_hash(tx)) + " included reference block that was too high");
```

Comment:
```suggestion
      "tx " << get_transaction_hash(tx) << " included reference block that was too high");
```

Slightly easier to read